### PR TITLE
added summary for the 2022-03 uPSC meeting

### PR DESCRIPTION
**Pull request by jonathanmtran**

### CODE REVIEW

File: governance/meetings/uportal-steering-committee-2022-03-09.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* Reviewed and moved things around as necessary
+
+### Open Apereo 2022
+
+Open Apereo 2022 is June 14 and 15. The uPSC discussed Open Apereo 2022 and the presentations members of the uPSC are considering submitting.
+See https://www.apereo.org/content/open-apereo-2022-value-open-source for more information regarding Open Apereo 2022
+
+### Editing rich content in uPortal
+
+The uPSC started the discussion regarding editing rich content in uPortal. There are many things to consider including: simple contnent, user interface components, and features/functionalitites of other content management systems.
```
Comment (ChristianMurphy) [inline governance/meetings/uportal-steering-committee-2022-03-09.md:21]:
```suggestion
The uPSC started the discussion regarding editing rich content in uPortal. There are many things to consider including: simple contnent, user interface components, and features/functionalitites of other content management systems.
```
```suggestion
The uPSC started the discussion regarding editing rich content in uPortal. There are many things to consider including: simple content, user interface components, and features/functionalities of other content management systems.
```

**Review by ChristianMurphy — APPROVED** [review]

File: governance/meetings/uportal-steering-committee-2022-03-09.md (hunk 1 above)
Comment (jonathanmtran) [reply governance/meetings/uportal-steering-committee-2022-03-09.md]: Thanks for catching that

File: governance/meetings/uportal-steering-committee-2022-03-09.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Attendees
+
+Jim Helwig, Chris Beach, Jonathan Tran, Julien Gribonvald, Benito Gonzalez
+
+## Topics
+
+### Task board updates
+
+* Got an update from Chris Beach regarding Playright and Artillery to do functional and load testing, respectively.
```
Comment (jimhelwig) [inline governance/meetings/uportal-steering-committee-2022-03-09.md:11]:
```suggestion
* Got an update from Chris Beach regarding Playwright and Artillery to do functional and load testing, respectively.
```

**Review by jimhelwig — APPROVED** [review]

File: governance/meetings/uportal-steering-committee-2022-03-09.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+### Task board updates
+
+* Got an update from Chris Beach regarding Playwright and Artillery to do functional and load testing, respectively.
+* Reviewed and moved things around as necessary
+
+### Open Apereo 2022
+
+Open Apereo 2022 is June 14 and 15. The uPSC discussed Open Apereo 2022 and the presentations members of the uPSC are considering submitting.
+See https://www.apereo.org/content/open-apereo-2022-value-open-source for more information regarding Open Apereo 2022
```
Comment (cbeach47) [inline governance/meetings/uportal-steering-committee-2022-03-09.md:17]:
```suggestion
See https://www.apereo.org/content/open-apereo-2022-value-open-source for more information regarding Open Apereo 2022.
```

**Review by cbeach47 — APPROVED** [review]
Minor comment.  Looks good!  👍

**Comment by jonathanmtran** [comment]
Thanks for the feedback, all

**Pull request merged**